opencga-server/src/main/java/org/opencb/opencga/server/rest/CohortWSServer.java

```diff
@@ -360,7 +361,8 @@ public Response deleteList(
 
     @GET
     @Path("/{cohorts}/acl")
-    @ApiOperation(value = "Return the acl of the cohort. If member is provided, it will only return the acl for the member.", response = Map.class)
+    @ApiOperation(value = "Return the acl of the cohort. If member is provided, it will only return the acl for the member.",
```

Comment:
Changing the return type is a breaking change... This might have implications for IVA and other clients.
This can't happen on a patch

Reply:
It is OK to change the type, but only if the outcome is compatible. i.e. only if the new output adds fields

Reply:
Also, as you are changing the API, you should have regenerated the CLI

Reply:
Yes, we need to regenerate the CLI. Also, I am synchronising with IVA developers so these changes will be applied at the same time.